tools/fm-editor/tree/FeatureTreeItem.cpp

```diff
@@ -0,0 +1,101 @@
+//
+// Created by simon on 02.02.23.
+//
+#include "FeatureTreeItem.h"
+#include <QMenu>
+#include <unordered_set>
+#include "../Utils.h"
+QVariant numericValue(vara::feature::Feature* Item) {
```

Comment:
```suggestion
#include "FeatureTreeItem.h"
#include "../Utils.h"

#include <QMenu>

#include <unordered_set>

QVariant numericValue(vara::feature::Feature* Item) {
```

PS: includes with `..` can be problematic, you should adapt this an correctly model the include dirs in the cmake so this header can be found.